apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -31,13 +35,108 @@ public void SimplestLog()
             var expectedJson = "{\"message\":\"test\",\"severity\":\"INFO\"}\n";
             var actualJson = writer.ToString();
 
-            Assert.Equal(actualJson, expectedJson);
+            Assert.Equal(expectedJson, actualJson);
+        }
+
+        [Fact]
+        public void InvalidLogLevel_Throws()
+        {
+            var formatter = CreateFormatter();
+            var logEntry = new LogEntry<string>((LogLevel)8, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
+            var writer = new StringWriter { NewLine = "\n" };
+
+            Assert.Throws<ArgumentOutOfRangeException>(() => formatter.Write(logEntry, scopeProvider: null, writer));
+        }
+
+        [Fact]
+        public void NoMessageAndNoException_Nothing()
+        {
+            var formatter = CreateFormatter();
+            var logEntry = new LogEntry<string>((LogLevel)8, "LogCategory", new EventId(1), null, exception: null, (state, exception) => state);
+            var writer = new StringWriter { NewLine = "\n" };
+            formatter.Write(logEntry, scopeProvider: null, writer);
+
+            var expectedJson = "";
+            var actualJson = writer.ToString();
+
+            Assert.Equal(expectedJson, actualJson);
+        }
+
+        [Fact]
+        public void Log_CorrectScopeInformation()
+        {
+            var formatter = CreateFormatter(new GoogleCloudConsoleFormatterOptions() { IncludeScopes = true });
+
+            var logEntry = new LogEntry<string>(LogLevel.Information, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
+            var writer = new StringWriter { NewLine = "\n" };
+
+            var scopeProvider = new LoggerExternalScopeProvider();
+
+            formatter.Write(logEntry, scopeProvider: scopeProvider, writer);
+            var expectedJson = "{\"message\":\"test\",\"severity\":\"INFO\",\"scopes\":[]}\n";
+            var actualJson = writer.ToString();
+
+            Assert.Equal(expectedJson, actualJson);
+        }
+
+
+        [Fact]
+        public void Log_KeyValuePair()
+        {
+            var formatter = CreateFormatter();
+
+            var _state = new ReadOnlyCollection<KeyValuePair<string, object>>(new List<KeyValuePair<string, object>>()
+            {
+                new KeyValuePair<string, object>("A","Sample value 1"),
+                new KeyValuePair<string, object>("B","Sample value 2"),
+            });
```

Comment:
```suggestion
            var _state = new ReadOnlyCollection<KeyValuePair<string, object>>(new List<KeyValuePair<string, object>>
            {
                new KeyValuePair<string, object>("A", "Sample value 1"),
                new KeyValuePair<string, object>("B", "Sample value 2"),
            });
```

Formatting, and removing parentheses.